src/scala/io/bazel/rules_scala/jmh_support/BenchmarkGenerator.scala

```diff
@@ -41,15 +41,24 @@ object BenchmarkGenerator {
     classPath: List[Path]
   )
 
+  private case class GenerationException(messageLines: Seq[String])
+    extends RuntimeException(messageLines.mkString("\n"))
+
   def main(argv: Array[String]): Unit = {
     val args = parseArgs(argv)
-    generateJmhBenchmark(
-      args.generatorType,
-      args.resultSourceJar,
-      args.resultResourceJar,
-      args.inputJar,
-      args.classPath
-    )
+    try {
+      generateJmhBenchmark(
+        args.generatorType,
+        args.resultSourceJar,
+        args.resultResourceJar,
+        args.inputJar,
+        args.classPath
+      )
+    } catch {
+      case GenerationException(messageLines) =>
```

Comment:
What happens if you don’t catch the exception?

Reply:
Same thing, but with a giant, unpleasant stack trace afterwards.